### PR TITLE
Implementation of projections on linear spaces

### LINKED ISSUE #11: Implementation of projections on linear spaces

### PR DESCRIPTION
**Pull request by Gczmy**
### **About**
----
The implementation of projections on linear spaces.

This is work in progress.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import raocp as r
 
+<<<<<<< HEAD
```
Comment (alphaville) [inline main.py]: resolve the conflicts

File: raocp/core/scenario_tree.py [hunk 2]
```diff
@@ -1,5 +1,107 @@
+<<<<<<< HEAD
```
Comment (alphaville) [inline raocp/core/scenario_tree.py]: this is a conflict - please resolve it

**Review by alphaville — CHANGES_REQUESTED** [review]
I don't see any code related to the computation of projections on linear spaces...

**Comment by Gczmy** [comment]
> I don't see any code related to the computation of projections on linear spaces...
I have uplode the "kernel.py" into folder "playground" and resove these conflicts to keep same with the branch "main".

**Review by alphaville — APPROVED** [review]

**Review by ruairimoran — APPROVED** [review]

**Pull request merged**